notifier/notifier.go

```diff
@@ -160,6 +180,13 @@ func (n *Notifier) ApplyConfig(conf *config.Config) error {
 		if err != nil {
 			return err
 		}
+
+		for _, am := range ams.ams {
```

Comment:
I'm assuming you just want to ensure the existence of metrics here without observing a value - interesting pattern. Maybe add a comment to that effect?